session/session.go

```diff
@@ -203,6 +203,21 @@ func (manager *Manager) getSid(r *http.Request) (string, error) {
 	return url.QueryUnescape(cookie.Value)
 }
 
+// Get session store from this http request,
```

Comment:
warning: comment on exported method Manager.CurrentSession should be of the form "CurrentSession ..." (golint)